cmd/logs.go

```diff
@@ -34,10 +34,11 @@ var LogsCmd = &cobra.Command{
 }
 
 func init() {
-	LogsCmd.Flags().BoolVarP(&k8s, "kubernetes", "k", true, "only works with a Kubernetes cluster")
-	LogsCmd.Flags().StringVarP(&logsAppID, "app-id", "a", "", "The app id for which logs are needed")
-	LogsCmd.Flags().StringVarP(&podName, "pod-name", "p", "", "(optional) Name of the Pod. Use this in case you have multiple app instances (Pods)")
-	LogsCmd.Flags().StringVarP(&namespace, "namespace", "n", "", "(optional) Kubernetes namespace in which your application is deployed. default value is 'default'")
+	LogsCmd.Flags().BoolVarP(&k8s, "kubernetes", "k", true, "Gets logs from a Kubernetes cluster")
+	LogsCmd.Flags().StringVarP(&logsAppID, "app-id", "a", "", "The application's id for which logs are needed")
+	LogsCmd.Flags().StringVarP(&podName, "pod-name", "p", "", "Name of the Pod in Kubernetes, in case your application has multiple pods (optional)")
+	LogsCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "Kubernetes namespace in which your application is deployed")
```

Comment:
Again the default value is missing. These are best kept. Like this.
Kubernetes namespace in which your application is deployed (default is "default")

Reply:
That is auto-generated by the library, here is how it looks at the command output:

  -n, --namespace string   Kubernetes namespace in which your application is deployed (default "default")